lib/App/index.js

```diff
@@ -679,12 +679,19 @@ class App {
    * @private
    */
   async _getPrunePaths() {
+    if (await NpmCommands.isNpmInstalled() === false) {
+      return;
```

Comment:
maybe trow an error? "npm is not installed!"

Reply:
I believe its intentional so the CLI works even if someone doesn't use npm...
I'm not sure how someone would install `node` and not have `npm`.
Additionally you need `node` to even run the CLI so it doesn't really make much sense IMO.